Source/JavaScriptCore/bytecode/Repatch.cpp

```diff
@@ -87,6 +87,9 @@ void linkMonomorphicCall(VM& vm, JSCell* owner, CallLinkInfo& callLinkInfo, Code
     CodeBlock* callerCodeBlock = jsDynamicCast<CodeBlock*>(owner); // WebAssembly -> JS stubs don't have a valid CodeBlock.
     ASSERT(owner);
 
+    if (Options::forceICFailure())
```

Comment:
This was helpful in testing when I added logging to the slow path

Reply:
Put `UNLIKELY` here.